LibreNMS/OS/Ray.php

```diff
@@ -117,9 +136,10 @@ public function discoverWirelessRssi()
      */
     public function discoverWirelessSnr()
     {
-        $oid = '.1.3.6.1.4.1.33555.1.3.2.2'; // RAY-MIB::snr.0
         return array(
-            new WirelessSensor('snr', $this->getDeviceId(), $oid, 'racom', 1, 'CINR', null, 1, 10),
+            new WirelessSensor('snr', $this->getDeviceId(), '.1.3.6.1.4.1.33555.1.3.2.2.0', 'racom', 1, 'CINR', null, 1, 10),
+            new WirelessSensor('snr', $this->getDeviceId(), '.1.3.6.1.4.1.33555.1.3.2.2', 'racom-old', 1, 'CINR', null, 1, 10),
```

Comment:
renaming the existing sensor types will cause data loss (it changes the rrd name)

Reply:
I'll fix this